scripts/pre_commit_linter.py

```diff
@@ -100,6 +100,18 @@
 }
 
 BAD_PATTERNS_JS_REGEXP = [
+    {
+        'regexp': r'\b(browser.explore)\(',
+        'message': "Please remove browser.explore() calls from e2e tests.",
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': r'\b(browser.pause)\(',
+        'message': "Please remove browser.pause() calls from e2e tests.",
```

Comment:
Similarly here: `In tests, please do not use browser.pause().`

Reply:
Done!